docs/advanced/mbed_targets.md

```diff
@@ -18,15 +18,20 @@ mbed uses JSON as a description language for its build targets. The JSON descrip
         "detect_code": ["0230"]
 ```
 
-The definition of the target called **TEENSY3_1** is a JSON object. The properties in the object are either "standard" (understood by the mbed build system) or specific to the target.
+The style that we use for targets is:
+ - Each property is on its own line.
+ - The open brace `{` is on the same line as the target name, or the property name that is its key.
+ - The close brace `}` or close brace and comma `},` are on their own line.
+ - Lists take up at most one line.
+ - Lines are not split at any number of columns.
```

Comment:
Normally, you would split long lines at some reasonable number of characters. 80 is common. Our targets.json file has no limit, and the style is not to split long lines.